tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/xa/BasicXaRecoveryTest.java

```diff
@@ -101,7 +101,7 @@ public void setUp() throws Exception {
       addressSettings.clear();
 
       if (storeType == StoreConfiguration.StoreType.DATABASE) {
-         configuration = createDefaultJDBCConfig().setJMXManagementEnabled(true);
```

Comment:
Not sure what you mean.  I added boolean to createDefaultJDBCConfig to allow the test to pass in whether or not t configure the server to set up a netty connector (previously it always used Netty). It should be using JDBC for the test.